module/app.js

```diff
@@ -16,5 +16,19 @@ module.getLocalStorageByKey = async ( key )  => {
 	})
 }
 
+module.setLocalStorageData = async ( key, value )  => {
+	return new Promise(( resolve, reject ) => {
+		try {
+			let obj = {};
+			obj[key] = value;
+			chrome.storage.sync.set( obj, ( result ) => {
```

Comment:
You don't have to assign it to an object, since its one time used you can directly write like this 🙂 
```suggestion
			chrome.storage.sync.set({ [key]: value }, ( result ) => {
```